lib/opentelemetry/context.lua

```diff
@@ -1,5 +1,7 @@
+local otel_global = require("opentelemetry.global")
```

Comment:
Need to make sure that this will be the same instance when required in different files.

Reply:
It's same instance in one nginx worker process.